Core/Sources/Cart/ShoppingCart.swift

```diff
@@ -339,7 +339,8 @@ public final class ShoppingCart: Codable {
         }
         self.items = newItems
         self.save(postEvent: false)
-        CartEvent.cart(self)
+        // APPS-491: Only fire cart event on user interaction
+        // CartEvent.cart(self)
```

Comment:
Warum willst du es nicht komplett entfernen? Es ist in der Git History vorhanden und verbunden zum Ticket, damit wir wissen warum es weg ist.

Reply:
00912ae39875ddf98954abd69c4d629c906ae89f